doc/source/tutorial-quickstart-huggingface.rst

```diff
@@ -1,229 +1,423 @@
 .. _quickstart-huggingface:
 
+###########################
+ Quickstart 🤗 Transformers
+###########################
+
+In this federated learning tutorial we will learn how to train a large
+language model (LLM) on the `IMDB
```

Comment:
I wonder if its better to keep it as "language model"

Reply:
It's based on BERT, and in recognition of [it's origins](https://en.wikipedia.org/wiki/BERT_(language_model)), I'm thinking to include it.

Reply:
ok, sounds fair the way it's framed in the article _"an early example of large language model"_. Let's keep it 👍